jellyfin_mpv_shim/player.py

```diff
@@ -149,7 +149,7 @@ def __init__(self):
                 {
                     "start_mpv": settings.mpv_ext_start,
                     "ipc_socket": settings.mpv_ext_ipc,
-                    "mpv_location": settings.mpv_ext_path,
```

Comment:
It should be that if the setting is not set and the platform is Mac OS it uses that path.

Reply:
That's a great idea. I'll look into how to do that. As an added note, the mpv executable is still runnning when the program is killed so that's another thing I have to take a look at

Reply:
Alright I've added that check but it's not a particularly clean solution...

Reply:
Minor complaint but can this be a variable outside of this config object?

Reply:
yeah I don't see why not